modules/vulnerability-management-scheduled-report.adoc

```diff
@@ -9,21 +9,21 @@
 The following procedure creates a scheduled vulnerability report.
 
 .Procedure
-. On the {product-title-short} portal, navigate to *Vulnerability Management* -> *Reporting*.
+. On the {product-title-short} portal, navigate to *Vulnerability Management 1.0* -> *Reporting*.
 . Click *Create report*.
 . Enter a name for your report in the *Report name* field.
 . Select a weekly or monthly cadence for your report from the *Repeat report...* drop-down list.
 . Select the days of the week for the report from the *On...* drop-down list.
 . Optional: Enter text describing the report in the *Description* field.
 . In the *CVE fixability type* field, select the common vulnerabilities and exposure (CVE) fixability types that you want to include in the report.
-. In the *Show vulnerabilities* drop-down list, select whether you want to show all vulnerabilities or show only vulnerabilities discovered since the last successful report.
-. In the *CVE severities* drop-down list, select the severities of the CVEs that should be included in the report. 
+. In the *Show vulnerabilities* drop-down list, select whether you want to show all vulnerabilities or show only vulnerabilities discovered since the last successful scheduled report.
+. In the *CVE severities* drop-down list, select the severities of the CVEs that should be included in the report.
 . In the *Configure report scope* field, select an existing collection, or click *Create collection* to create a new one. Entering text in the field searches for collections matching that text string. For more information about collections, see "Creating deployment collections" in the Additional resources section.
```

Comment:
Is "Creating deployment collections" supposed to be titled "Creating and using deployment collections"? Also add quotes around "Additional resources" here to match style elsewhere.